src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-lenght--31 calendar--start-day--mon;">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--mon">
```
Semicolon is redundant in class name. Also, there is a typo in `length`